lib/yarp.rb

```diff
@@ -95,7 +95,7 @@ def start_column
     # The column number in bytes where this location ends from the start of the
     # line.
     def end_column
-      source.column(end_offset - 1)
+      source.column(end_offset)
     end
```

Comment:
It would be good to document the `end_column` is now exclusive (like `Range#exclude_end?`).

FWIW this seems consistent with `RubyVM::AbstractSyntaxTree::Node.last_column`:
```
irb(main):001:0> node=RubyVM::AbstractSyntaxTree.parse("class Foo\nend").children[2]
=> (CLASS@1:0-2:3 (COLON2@1:6-1:9 nil :Foo) nil (SCOPE@1:0-2:3 tbl: [] args: nil body: (BEGIN@1:9-1:9 nil)))
irb(main):002:0> node.last_column
=> 3
irb(main):003:0> node.last_lineno
=> 2
```